Mathlib/Algebra/Module/LocalizedModule/Basic.lean

```diff
@@ -150,9 +150,7 @@ theorem mk_add_mk {m1 m2 : M} {s1 s2 : S} :
   mk_eq.mpr <| ⟨1, rfl⟩
 
 private theorem add_assoc' (x y z : LocalizedModule S M) : x + y + z = x + (y + z) := by
-  induction' x with mx sx
-  induction' y with my sy
-  induction' z with mz sz
+  cases x; cases y; cases z; rename_i mx sx my sy mz sz
```

Comment:
`rename_i` seems like a step backwards.

Reply:
It saves space and is in my opinion more readable than having three `cases x with | _ mx sx => ?_`.

Reply:
We aren't saving lines of code. With `rename_i`, it isn't obvious where the variables come from.